src/ErrorHandling/Breakpoint.cpp

```diff
@@ -0,0 +1,26 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "ErrorHandling/Breakpoint.hpp"
+
+#include <csignal>
+
+namespace {
+void (*sig_ignore)(int) = SIG_IGN; // NOLINT
```

Comment:
Inline this.  It was split out in SpEC because the SIG_IGN macro caused a compiler warning and putting the warning pragmas inside the function didn't work in some old compiler we were using.